test/CreatorToken.t.sol

```diff
@@ -484,3 +497,51 @@ contract Pausing is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract bondingCurveTests is CreatorTokenTest {
+  function test_BuyPriceIsCorrect(address _buyer) public {
+    vm.assume(
+      _buyer != address(0) && _buyer != address(creatorToken) && _buyer != creator
+        && _buyer != admin
+    );
+    uint256 buyNTokens = 5;
+
+    for (uint256 i = 0; i < buyNTokens; i++) {
```

Comment:
Nit: the `i` should follow the `_` naming convention.

Reply:
Updated this in the same commit as rest of the naming updates: https://github.com/showtime-xyz/creator-tokens/commit/89f1666530442849e5fcc6a35261447ee2cf1cbf